src/app/banner/page.tsx

```
@@ -0,0 +1,156 @@
+          className="text-gray-50 absolute top-20 right-8 cursor-pointer"
+          onClick={() => {
+            navigator.clipboard.writeText(JSON.stringify(props, null, 2));
+          }}
+          title="Copy props to clipboard"
+        >
+          <Copy />
+        </button>
+        <h2 className="font-medium text-xl mb-4">Props</h2>
+        <p
+          className="flex flex-col w-full min-w-[300px] bg-gray-100 h-[90%] overflow-auto p-2"
+          dangerouslySetInnerHTML={{ __html: sanitizeHtml(htmlProps) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)